bx_py_utils/test_utils/unittest_utils.py

```diff
@@ -20,3 +26,36 @@ def assert_no_flat_tests_functions(path: Path):
 
     if errors:
         raise AssertionError(f'Flat test files found:\n{pprint.pformat(errors)}')
+
+
+class BaseDocTests(TestCase):
+    """
+    Helper to include all doctests in unittests, without change unittest setup. Just add a normal TestCase.
```

Comment:
We should add a note here: This is extremely slow, and can easily take as long as all other tests combined for many projects!

Reply:
This will take some time. But it's IMHO fast and not extremely slow.
The bx_py_utils tests takes 1,47sec and the doctest part 0.0225sec. This is not slow, i think.